Add tswilliamson's calctype utility

dr-carlos opened the pull request:
This PR adds tswilliamson's calctype utility (straight from the PrizmSDK).
It compiles fine, and the prebuilt exe opens (but I can't test anymore because WINE doesn't work well on my machine).

tari commented:
Seems reasonable, but a few not-so-small thoughts:

 * we shouldn't check in binaries, especially not .exe ones (even if it is a .NET binary and not a straight Windows binary)
 * an example in the examples directory would be nice (even if it's not very complex)

The first is probably most difficult. As it's a C# application it probably wouldn't be hard to make it build with the newfangled .NET Core and we can [distribute standalone binaries for various platforms](https://learn.microsoft.com/en-us/dotnet/core/deploying/#publish-self-contained), but I also think the tool should avoid being a clicky GUI app unless there's a really compelling reason. Though since this also depends on [BMFont](http://www.angelcode.com/products/bmfont/) that seems like it only supports Windows, it's probably best to omit the PC tool and simply point users at where to get it elsewhere if they want to use fonts other than the provided ones.

dr-carlos commented:
I've removed BMFont and the gui.
I also added UTILS_INC to prizm_rules so it can be easily included in Makefiles.

Lastly I fixed up some bugs and added an example.

tari commented:
Cool, thanks! That example is very helpful for me to understand how this is useful. :)
